frontend/src/pages/ParentsPage.js

```diff
@@ -13,49 +19,30 @@ class ParentsPage extends Component {
     }
   ];
 
-  adviceCards = [
-    {
-      title: 'An Introduction',
-      subtitle:
-        'Philadelphia reads is raising a city of readers for a better tomorrow!',
-      text:
-        'The mission of Philadelphia READS is to “raise a city of readers” through quality out of school time programs focused to get children and youth to read on or above grade level by grade 4.',
-      smallText: '',
-      button: 'More',
-      image: philareads1
-    },
-    {
-      title: 'Need Advice?',
-      subtitle: "Here's our first tip!",
-      text:
-        "Being a part of your child's learning process is very effective. For example, read aloud with them and ask questions about stories as you read along. this will promote critical thinking and recollection quickly.",
-      smallText: 'Last updated 3 mins ago',
-      button: 'More',
-      image: philareads2
-    },
-    {
-      title: 'Health Matters Too!',
-      subtitle:
-        'Reading every day will help your child get better and better over time, however health is also important!',
-      text:
-        "A healthy diet, exercise routine, and sleep schedule will help your child's brain be ready to intake more information and do some heavy brain lifting 😃",
-      button: 'More',
-      image: philareads3
-    },
-    {
-      title: 'Example',
-      subtitle: 'This example has no buttons or corresponding images.',
-      text: "That's okay though. Images aren't necessary for making a card 🙂",
-      smallText: ''
+  fetchAdviceData = async () => {
+    const { message, success, result } = await getAdvice(
+      this.props.match.params.id
+    );
+    if (success) {
+      if (result['results'].length > 0) {
+        this.setState({ adviceData: result['results'][0]['text'] });
```

Comment:
oh yea just need to change adviceData to an array and get all the elements from the results array